src/ts/langs/en_GB.ts

```diff
@@ -24,6 +24,10 @@ const t = {
     "tag-delete-warning": "Delete this tag?",
     "template-title": "Template title",
     "upload-file-comment": "File comment",
+    "step-added": "Step added",
```

Comment:
I'm wary of adding new terms to translate. I think here having "Saved" or nothing is enough. 4 terms times 16 languages is 64 strings that need work to translate.